google-beta/resource_compute_disk.go

```diff
@@ -1198,16 +1274,17 @@ func resourceComputeDiskEncoder(d *schema.ResourceData, meta interface{}, obj ma
 }
 
 func resourceComputeDiskDecoder(d *schema.ResourceData, meta interface{}, res map[string]interface{}) (map[string]interface{}, error) {
+	// The response for crypto keys often includes the version of the key which needs to be removed
+	// format: projects/<project>/locations/<region>/keyRings/<keyring>/cryptoKeys/<key>/cryptoKeyVersions/1
+	reg := regexp.MustCompile("/cryptoKeyVersions")
```

Comment:
since this is for the whole decoder, I'd probably call it something a bit closer related to what it actually is- even ckv would work.

also does this need to be a regex. would a strings.Split work too?